recipes/ipyparallel/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "5.0.1" %}
+
+package:
+  name: ipyparallel
+  version: {{ version }}
+
+source:
+  fn: ipyparallel-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipyparallel/ipyparallel-{{ version }}.tar.gz
+  md5: e771add417b0d1b98d3ae92099becce5
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
We have been adding other options like `--single-version-externally-managed --record record.txt` when using packages with `setuptools`, which has worked nicely so far. Though you do make a good point as `pip install --no-deps .` would avoid us having to check this at all. However, leaving `--no-deps` off could be ok as this will raise red flags if we missed adding a dependency as `conda` terminates any build if it tries to download requirements with `pip`.